src/Control/Lens/At.hs

```diff
@@ -454,17 +458,25 @@ instance At (Maybe a) where
   {-# INLINE at #-}
 
 instance At (IntMap a) where
+#if MIN_VERSION_containers(0,5,8)
```

Comment:
I'm not sure there's much point to this one right now. The definion of `Data.IntMap.alterF` was lifted straight from here! Someone (either `containers` or `lens`) should add some `RULES` to specialize to `Identity` and `Const`. I suppose that probably should be `containers`, at which point this change is justified. I don't really care *too* much either way. Do you have strong feelings about it?